setup/event_track_ticket/.eggs/README.txt

```diff
@@ -0,0 +1,6 @@
+This directory contains eggs that were downloaded by setuptools to build, test, and run plug-ins.
```

Comment:
@tschanzt pls remove `setup` folder: it won't hurt as it's going to be re-generated but is pointless for reviewing